scripts/publish.ts

```diff
@@ -9,7 +9,7 @@ shell.cp(['LICENSE'], 'packages/@wdio_electron-utils');
 shell.cp(['LICENSE'], 'packages/@wdio_electron-types');
 
 // --no-git-checks is used to skip the git checks - due to getting erroneous ERR_PNPM_GIT_UNCLEAN errors
-const publishCommand = `pnpm publish -r --no-git-checks ${options.join(' ')}`;
+const publishCommand = `pnpm publish -r --access=public --no-git-checks ${options.join(' ')}`;
```

Comment:
```suggestion
const publishCommand = `pnpm publish -r --access public --no-git-checks ${options.join(' ')}`;
```

https://docs.npmjs.com/creating-and-publishing-scoped-public-packages#publishing-scoped-public-packages